src/dbcbs_utils.hpp

```diff
@@ -676,6 +676,45 @@ void export_solutions_joint(const std::vector<LowLevelPlan<dynobench::Trajectory
       *out << joint_actions.at(j).format(dynobench::FMT)<< std::endl;
   }
 }
+
+void extract_motion_primitives(dynobench::Problem problem,
+                              MultiRobotTrajectory &multi_robot_opt_out,
+                              std::map<std::string, std::vector<dynoplan::Motion>> &robot_motions,
+                              const std::vector<std::shared_ptr<dynobench::Model_robot>> &all_robots){
+  
+  size_t robot_idx = 0;
+  for(auto robot_trajectory : multi_robot_opt_out.trajectories) {
+    int idx = 0;
+    int total_actions = robot_trajectory.actions.size();
+    while (idx < total_actions){
+      int num_actions = rand() % 11 + 1; // [0-10]
+      num_actions = std::min(num_actions, (total_actions - idx));
+      if ((total_actions - (idx + num_actions)) < 5 && idx + num_actions < total_actions) {
```

Comment:
what's the magic 5?

Reply:
nothing magic, only ensures that the motions have >=5 length